### PR TITLE
{tools}[GCCcore/13.2.0] Z3 v4.13.0

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
n1285 - Linux RHEL 8.7 (Ootpa), x86_64, Intel(R) Xeon(R) Platinum 8470 (icelake), Python 3.8.13
See https://gist.github.com/Flamefire/4598c6fd487617459461b41414a427e0 for a full test report.

File: easybuild/easyconfigs/z/Z3/Z3-4.13.0-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+local_z3_site_path = "lib/python%(pyshortver)s/site-packages/%(namelower)s"
+postinstallcmds = [
+    'ln -s %s/include "%%(installdir)s/include"' % local_z3_site_path,
+    'cd "%%(installdir)s"; for lib in %s/lib/*; do ln -s ../$lib lib/$(basename $lib); done' % local_z3_site_path
+]
+
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': ['bin/z3', 'include/z3_api.h', 'lib/libz3.so'],
```
Comment (casparvl) [inline easybuild/easyconfigs/z/Z3/Z3-4.13.0-GCCcore-13.2.0.eb:46]: `SHLIB_EXT` instead of `.so` :)

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen3

**Review by casparvl — APPROVED** [review]
Lgtm!

**Comment by casparvl** [comment]
Going in, thanks @Flamefire!

**Pull request merged**